docs/CHANGELOG.rst

```diff
@@ -4,6 +4,19 @@ Change Log
 
 All notable changes to this project are documented in this file.
 
+===========
+Unreleased:
+===========
+
+Changed
+-------
+
+Fixed
+-----
+
+Added
+-----
+* MACS (v1.42) processor
```

Comment:
What about just `MACS14 processor` or `Processor for MACS14`? I'm guessing it is not tied to a specific version (v1.4.2) but a the whole series (v1.4.x).